src/common/prometheus/prometheus.service.ts

```diff
@@ -299,15 +300,21 @@ export class PrometheusService implements OnApplicationBootstrap {
     labelNames: ['nos_name'],
   });
 
+  public otherSyncParticipationAvgPercent = this.getOrCreateMetric('Gauge', {
+    name: METRIC_OTHER_SYNC_PARTICIPATION_AVG_PERCENT,
+    help: 'Other sync committee validators participation avg percent',
```

Comment:
_Others_ or smth otherwise looks confusing

Reply:
Established term in balval context (on dashboards, for example). If needed, I can make additional explanation for description. Something like `Other (not user monitored) sync committee validators ...`